tests/runtests.sh

```diff
@@ -9,4 +9,4 @@ emerge --sync
 emerge -q --buildpkg --usepkg dev-vcs/git app-portage/repoman
 
 # Run the tests
-repoman full --xmlparse
+repoman full -d
```

Comment:
From `man repoman`
```       -d, --include-dev
              Include dev profiles in dependency checks.
```
I never fully understood what a dev profile is. I thought this will call repoman on the dependencies as well.